R/readTimeseries.R

```diff
@@ -25,7 +28,25 @@ readTimeSeries <- function(series, con,
                            schema = "timeseries",
                            env = NULL,
                            pkg_for_irreg = "xts",
-                           chunksize = 10000){
+                           chunksize = 10000,
+                           regex = FALSE){
+  
+  if(regex) {
+    if(length(series) > 1) {
+      stop("Only supports a single expression in series!")
```

Comment:
This ain't true. Fuzzy searches can give a vector of keys back. Doesn't it make sense to return ALL series whose keys match?

Reply:
What I mean is that only a single string should be supplied in series if `regex == TRUE`. i.e.
```
readTimeSeries(c("ch\\.kof.*", "us\\.datastream.*"), con, regex = TRUE)
```
would be illegal.
That's an arbitrary decision though, we could also do `ts_key ~ '(ch\.kof.*|us\.datastream.*)'` in that case.